### DISCUSSION TITLE
Recompile Question

### DISCUSSION THREAD

**Discussion by ghost** [EricZimmerman/KapeFiles#707] [Q&A]
Team, is there a way to recompile KAPE to include the modules and targets folders populated with needed parts for each and executing the developed command calling KAPE.exe as on exe? Has anyone done this?

**Comment by EricZimmerman** [comment]
you mean so its a single exe? no, it doesnt work like that.

you can certainly delete targets and modules you do not want, but each still needs to be within the Targets or Modules folder for kape to find em

**Comment by ghost** [comment]
So, no way to make it a executable bundle that calls the kape command after unwrapping and firing the kape command it set to. Other ideas other than a zip with batch file?

**Comment by ghost** [comment]
The goal was basically this  example:

Forensic.exe(is the main Execution wrapper like a zip) ----> extracts Kape.exe, targets and modules ----------> then fires the .Kape xxxx command line set to call the unwrapped Kape to run using it.

**Comment by EricZimmerman** [comment]
you can certainly make an exe that does that for you, but its not something KAPE allows you to do in and of itself
